pkg/controllers/routing/ecmp_vip.go

```diff
@@ -47,6 +48,7 @@ func (nrc *NetworkRoutingController) advertiseVIPs(vips []string) {
 		if err != nil {
 			glog.Errorf("error advertising IP: %q, error: %v", vip, err)
 		}
+		metrics.ControllerBGPadvertisements.WithLabelValues("sent").Inc()
```

Comment:
does this need to check for `nrc.MetricsEnabled`?

Reply:
as we discussed on Slack i think we can just have these as they are as the metrics are instanciated even if metrics controller is disabled so no errors are triggered and we don't need to have 2 additional lines of code per metric checking if metrics are enabled

Reply:
i procceded and removed the extra if statements and config variable where it didn't make sense to have it any more. i left the one in the service controller since there is a function there that uses quiet alot of ipvs calls when doing sync just to get the metrics